esx_service/utils/auth.py

```diff
@@ -367,9 +370,9 @@ def tables_exist():
         return str(e), False
 
     if not result:
-        error_info = "table volumes does not exist"
-        logging.error(error_info)
-        return error_info, False
+        error_msg = "table volumes does not exist"
```

Comment:
nit: would be nicer to have a function `err_msg_no_table('volumes') and use everywhere